src/software/ai/hl/stp/play/enemy_ball_placement_play_test.cpp

```diff
@@ -37,10 +37,11 @@ TEST_F(EnemyBallPlacementPlayTest, test_ball_placement_center)
 
     std::vector<ValidationFunction> terminating_validation_functions = {
         [](std::shared_ptr<World> world_ptr, ValidationCoroutine::push_type& yield) {
-            while (world_ptr->getMostRecentTimestamp() < Timestamp::fromSeconds(9.5))
+            while (world_ptr->getMostRecentTimestamp() < Timestamp::fromSeconds(2))
             {
-                yield("Timestamp not at 9.5s");
+                yield("");
             }
+            robotHalt(world_ptr, yield);
         }};
 
     std::vector<ValidationFunction> non_terminating_validation_functions = {};
```

Comment:
we should check that robots don't violate any conditions

Reply:
I'm going to add that tomorrow